src/core_atmosphere/physics/mpas_atmphys_driver_sfclayer.F

```diff
@@ -1193,7 +1195,7 @@ subroutine driver_sfclayer(itimestep,configs,mesh,diag_physics,sfc_input,its,ite
  dx = sqrt(maxval(areaCell))
 
  initflag = 1
- if(config_do_restart .or. itimestep > 1) initflag = 0
+ if(config_do_restart .or. config_do_DAcycling .or. itimestep > 1) initflag = 0
```

Comment:
@tanyasmirnova Could you help me understand what `initflag` means?
0 means init from cold start?  1 means init from warm start? Thanks!

Reply:
initflag = 1 for cold-start, and initflag=0 for restart and DAcycling.

Reply:
Got it. Thanks!